src/adapters/postgres/user-adapter.ts

```
@@ -2966,6 +2969,263 @@ export class PostgresUserService implements IServicelocator {
+      const keycloakTokenResponse = await getKeycloakAdminToken();
+      const keycloakToken = keycloakTokenResponse.data.access_token;
+
+      // Process every user concurrently; a failure on one email must not block the others.
+      const settled = await Promise.allSettled(
+        emails.map((email) =>
+          this.anonymizeSingleUser(
+            email,
+            reason,
+            loggedInUserId,
+            keycloakToken
+          )
```

ESLint

[error] Replace `⏎············email,⏎············reason,⏎············loggedInUserId,⏎············keycloakToken⏎··········` with `email,·reason,·loggedInUserId,·keycloakToken`

(prettier/prettier)